endorsement/management/commands/validate_endorsees.py

```diff
@@ -31,13 +30,8 @@ def handle(self, *args, **options):
             endorsee__netid__in=netids, category_code__in=categories)
 
         if len(endorsements):
```

Comment:
Is this check necessary? I *think* that the filter query would return something that's Iterable even if it has a length of 0

Reply:
not really since the email between was removed